cmd/doc/main.go

```diff
@@ -62,11 +62,7 @@ var docTemplate = template.Must(template.New("doc.html").Funcs(
 			}
 		},
 	},
-).ParseFiles("template/doc.html", "template/analytics.html", "template/twitter.html"))
-
-var orgTemplate = template.Must(template.ParseFiles("template/org.html", "template/analytics.html", "template/twitter.html"))
-var newTemplate = template.Must(template.ParseFiles("template/new.html", "template/analytics.html", "template/twitter.html"))
-var homeTemplate = template.Must(template.ParseFiles("template/home.html", "template/analytics.html", "template/twitter.html"))
+).ParseGlob("template/*.html"))
```

Comment:
Thoughts on using something like https://github.com/unrolled/render to get layouts and de-dupe things a bit more?

Reply:
Not super familiar, but I am all for it if it simplifies some of the gross logic that currently exists in the template :sweat_smile:

Reply:
Noted! I'll give it a try